packages/annotations/package.json

```diff
@@ -26,6 +26,7 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/rich-text": "file:../rich-text",
 		"lodash": "^4.17.10",
+		"memize": "^1.0.5",
```

Comment:
I think we also need it on the server side.

Reply:
No we don't, there's no script for memize

Reply:
Oh, cool then :)